client/js/README.md

```diff
@@ -255,9 +255,9 @@ console.log(api_info);
 The `config` property contains the configuration for the gradio application you are connected to. This object may contain useful meta information about the application.
 
 ```ts
-import { client } from "@gradio/client";
+import { Client } from "@gradio/client";
 
-const app = await client("user/space-name");
+const app = await Client.create("user/space-name");
 console.log(app.config);
 ```
 
```

Comment:
At the bottom of this document, we discuss `duplicate`. I believe this has been refactored as well, correct?